docusaurus/docs/create-a-plugin/extend-a-plugin/add-logs-metrics-traces-for-backend-plugins.md

```diff
@@ -0,0 +1,421 @@
+
+Refer to the [OpenTelemetry Go SDK](https://pkg.go.dev/go.opentelemetry.io/otel) for in-depth documentation about all the features provided by OpenTelemetry.
+
+:::note
+
+If tracing is disabled in Grafana, `backend.DefaultTracer()` returns a no-op tracer.
+
+:::
+
+### Implement tracing in your plugin
+
+:::note
+
+Make sure you are using at least [`grafana-plugin-sdk-go v0.157.0`](https://github.com/grafana/grafana-plugin-sdk-go/releases/tag/v0.157.0). Refer to [Update the Go SDK](../develop-a-plugin/work-with-backend#update-the-go-sdk) for update instructions.
+
+:::
+
+#### Configure a global tracer
+
+When OpenTelemetry tracing is enabled on the main Grafana instance and tracing is enabled for a plugin, the OpenTelemetry endpoint address and propagation format is passed to the plugin during startup. These parameters are used to configure a global tracer.
+
+1. Use `datasource.Manage` or `app.Manage` to run your plugin to automatically configure the global tracer. Specify any custom attributes for the default tracer using `CustomAttributes`:
+
+   ```go
+   func main() {
+       if err := datasource.Manage("MY_PLUGIN_ID", plugin.NewDatasource, datasource.ManageOpts{
+           TracingOpts: tracing.Opts{
+               // Optional custom attributes attached to the tracer's resource.
+               // The tracer will already have some SDK and runtime ones pre-populated.
+               CustomAttributes: []attribute.KeyValue{
+                   attribute.String("my_plugin.my_attribute", "custom value"),
+               },
+           },
+       }); err != nil {
+           log.DefaultLogger.Error(err.Error())
+           os.Exit(1)
+       }
+   }
+   ```
+
+1. Once you have configured tracing, use the global tracer like this:
+
+   ```go
+   tracing.DefaultTracer()
+   ```
+
+   This returns an [OpenTelemetry `trace.Tracer`](https://pkg.go.dev/go.opentelemetry.io/otel/trace#Tracer) for creating spans.
+
+   **Example:**
+
+   ```go
+   func (d *Datasource) query(ctx context.Context, pCtx backend.PluginContext, query backend.DataQuery) (backend.DataResponse, error) {
+       ctx, span := tracing.DefaultTracer().Start(
+           ctx,
+           "query processing",
+           trace.WithAttributes(
+               attribute.String("query.ref_id", query.RefID),
+               attribute.String("query.type", query.QueryType),
+               attribute.Int64("query.max_data_points", query.MaxDataPoints),
+               attribute.Int64("query.interval_ms", query.Interval.Milliseconds()),
+               attribute.Int64("query.time_range.from", query.TimeRange.From.Unix()),
+               attribute.Int64("query.time_range.to", query.TimeRange.To.Unix()),
+           ),
+       )
+       defer span.End()
+
+       // ...
+   }
+   ```
+
+### Automatic instrumentation by the SDK
+
+The SDK automates some instrumentation to ease developer experience.
```

Comment:
```suggestion
The SDK automates some instrumentation to ease developer experience. This section explores the default tracing added to gRPC calls and outgoing HTTP requests. 
```

Just a suggestion, otherwise it feels like a throwaway comment